docs update (#1696)
hasura/graphql-engine: docs/graphql/manual/deployment/docker/index.rst

Step 2: Configure the **docker-run.sh** script
----------------------------------------------

> The ``docker-run.sh`` script has a sample docker run command in it. The following changes are needed to be

Reviewer: ```suggestion
The ``docker-run.sh`` script has a sample docker run command in it. The following changes have to be
```
